examples/mapping_host_file (for OS provisioning).csv

```diff
@@ -0,0 +1,3 @@
+MAC,Hostname,IP
+xx:yy:zz:aa:bb,server,1.2.3.4
+aa:bb:cc:dd:ee,server2,10.10.11.12
```

Comment:
Can we change the file name so that there are no spaces?

Reply:
Done